pallets/weights/src/pallet_settlement.rs

```diff
@@ -0,0 +1,99 @@
+            .saturating_add(DbWeight::get().reads(4 as Weight))
+            .saturating_add(DbWeight::get().reads((5 as Weight).saturating_mul(l as Weight)))
+            .saturating_add(DbWeight::get().writes((6 as Weight).saturating_mul(l as Weight)))
+    }
+    fn set_venue_filtering() -> Weight {
+        (129_793_000 as Weight)
+            .saturating_add(DbWeight::get().reads(5 as Weight))
+            .saturating_add(DbWeight::get().writes(1 as Weight))
+    }
+    fn set_venue_filtering_disallow() -> Weight {
+        (123_320_000 as Weight)
+            .saturating_add(DbWeight::get().reads(5 as Weight))
+            .saturating_add(DbWeight::get().writes(1 as Weight))
+    }
+    fn allow_venues(v: u32) -> Weight {
+        (95_877_000 as Weight)
+            .saturating_add((6_462_000 as Weight).saturating_mul(v as Weight))
+            .saturating_add(DbWeight::get().reads(5 as Weight))
+            .saturating_add(DbWeight::get().writes((1 as Weight).saturating_mul(v as Weight)))
+    }
+    fn disallow_venues(v: u32) -> Weight {
+        (93_907_000 as Weight)
+            .saturating_add((6_054_000 as Weight).saturating_mul(v as Weight))
+            .saturating_add(DbWeight::get().reads(5 as Weight))
+            .saturating_add(DbWeight::get().writes((1 as Weight).saturating_mul(v as Weight)))
+    }
+    fn withdraw_affirmation(l: u32) -> Weight {
+        (5_414_000 as Weight)
+            .saturating_add((103_665_000 as Weight).saturating_mul(l as Weight))
+            .saturating_add(DbWeight::get().reads(4 as Weight))
+            .saturating_add(DbWeight::get().reads((5 as Weight).saturating_mul(l as Weight)))
+            .saturating_add(DbWeight::get().writes((4 as Weight).saturating_mul(l as Weight)))
+    }
+    fn withdraw_affirmation_with_receipt(l: u32) -> Weight {
+        (10_246_000 as Weight)
+            .saturating_add((106_451_000 as Weight).saturating_mul(l as Weight))
+            .saturating_add(DbWeight::get().reads(4 as Weight))
+            .saturating_add(DbWeight::get().reads((4 as Weight).saturating_mul(l as Weight)))
+            .saturating_add(DbWeight::get().writes((4 as Weight).saturating_mul(l as Weight)))
+    }
+    fn withdraw_affirmation_with_both_receipt_and_onchain_affirmation(l: u32) -> Weight {
+        (0 as Weight)
+            .saturating_add((109_970_000 as Weight).saturating_mul(l as Weight))
+            .saturating_add(DbWeight::get().reads(3 as Weight))
+            .saturating_add(DbWeight::get().reads((4 as Weight).saturating_mul(l as Weight)))
+            .saturating_add(DbWeight::get().writes((4 as Weight).saturating_mul(l as Weight)))
+    }
+    fn unclaim_receipt() -> Weight {
+        (164_417_000 as Weight)
+            .saturating_add(DbWeight::get().reads(10 as Weight))
+            .saturating_add(DbWeight::get().writes(3 as Weight))
+    }
+    fn affirm_instruction() -> u64 {
+        todo!()
```

Comment:
It will never trigger as I am assigning weightInfo = () in the runtime trait definition, which does have the fixed placeholder value and in the transaction weight calculation we are using `weight_for` module functions to calculate the weight. I think it is ok for now to keep as it is once we have a benchmark for every extrinsic then we can think of it.